es67x/src/main/scala/tech/beshu/ror/es/IndexLevelActionFilter.scala

```diff
@@ -60,7 +60,7 @@ class IndexLevelActionFilter(clusterService: ClusterService,
                              esInitListener: EsInitListener,
                              rorEsConfig: ReadonlyRestEsConfig)
                             (implicit environmentConfig: EnvironmentConfig)
-  extends ActionFilter with Logging {
+  extends ActionFilter with RequestIdAwareLogging {
```

Comment:
The changes in modules are 99% the same, with some minor module-specific differences